### DISCUSSION TITLE
Cinnamon in as my chroot!

### DISCUSSION THREAD

**Discussion by Arvin2A** [dnschneid/crouton#4593] [Ideas]
Can anyone one have ideas how to re-make cinnamon using this site tutorial :https://www.linuxuprising.com/2019/01/how-to-install-cinnamon-desktop-on.html I just used the tutorial web site to get a cinnamon chroot

**Comment by DennisLfromGA** [comment]
@Arvin2A,

I haven't tried it but that tutorial looks to be pretty complete and easy to understand.

You stated:
> I just used the tutorial web site to get a cinnamon chroot

Did it not work, please explain???   

You asked:
> Can anyone one have ideas how to re-make cinnamon using this site tutorial  

What exactly do you mean by 'remmake cinnamon'???

-**DennisLfromGA**

**Reply by Arvin2A** [reply]
i mean getting back the ''cinnamon'' target, i am pretty new to crouton. The site worked.

**Comment by DennisLfromGA** [comment]
@Arvin2A,

You cannot exactly get back the 'cinnamon target', you or someone else will have to make a 'cinnamon target' and then file a 'pull request' and let the crouton devs review & approve it, once that's done they'll merge it for others to use.

Since the crouton devs have to test the proposed target on each release along with different SoC's like Intel, ARM, etc. it's a much harder thing to do than it sounds like. 

It was removed because the PPA went away - https://github.com/dnschneid/crouton/pull/832

*Hope this helps,*
-**DennisLfromGA**

**Reply by Arvin2A** [reply]
_no wayyyyyyyyyyyy he still has that profile picture_

Hi I destroyed my chromebook i have arch linux now i was a dum 3rd grader back then haha

**Reply by Arvin2A** [reply]
how are you doing
